test/local.sh

```diff
@@ -0,0 +1,11 @@
+if [ -z "$root" ]; then
+	echo 'ERROR: `$root` is undefined'
+	exit 1
+fi
+
+faucet="require('$root/test/hooks.js'); require('$root/node_modules/.bin/faucet');"
+
+function faucet {
+	args="$@"
+	node -e "$faucet" -- "" "$@"
+}
```

Comment:
is it worth extracting these two files into a common package (e.g. faucet-pipeline-util)? Perhaps even some of the functionality iof `test/run`?